test/signalrclienttests/logger_tests.cpp

```diff
@@ -61,9 +61,11 @@ TEST(logger_write, entries_formatted_correctly)
 
     auto entry = log_entries[0];
 
-    auto date_str = entry.substr(0, entry.find_first_of("Z") + 1);
-    auto date = utility::datetime::from_string(utility::conversions::to_string_t(date_str), utility::datetime::ISO_8601);
-    ASSERT_EQ(date_str, utility::conversions::to_utf8string(date.to_string(utility::datetime::ISO_8601)));
+    //0000-0-00T00:00:00Z [message     ] message\n
+    std::regex r{ "[\\d]{4}-[\\d]{2}-[\\d]{2}T[\\d]{2}:[\\d]{2}:[\\d]{2}Z \\[.{12}\\] \\w{1,}\\n" };
```

Comment:
Could you just put ` \\[message     \\] message\\n` instead of the more generic ` \\[.{12}\\] \\w{1,}\\n` at the end of the regex and do both parts of the test at once?

Reply:
I'll try, and if you have other ideas for cleaning this regex up that'd be nice too :D